src/features/content-explorer/content-explorer/ContentExplorerActionButtons.js

```diff
@@ -46,9 +47,13 @@ const ContentExplorerActionButtons = ({
             chosenItems = [currentFolder];
         }
 
-        if (onChooseClick && chosenItems.length > 0) {
+        if (onChooseClick && (chosenItems.length > 0 || allowChoosingEmptySelection)) {
             onChooseClick(chosenItems);
         }
+
+        // if (onChooseClick && chosenItems.length > 0) {
+        //     onChooseClick(chosenItems);
+        // }
     };
 
```

Comment:
I think its safe to say that we can delete these comments

Reply:
haha yes, thank you